org.kde.kanagram.json

```diff
@@ -0,0 +1,49 @@
+{
+    "id": "org.kde.kanagram",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "rename-icon": "kanagram",
+    "sdk": "org.kde.Sdk",
+    "command": "kanagram",
+    "finish-args": ["--share=network", "--device=dri", "--socket=pulseaudio", "--share=ipc", "--socket=x11", "--socket=wayland" ],
+
```

Comment:
You should add `cleanup` array with at least `/include` and `/lib/cmake` to save some space.